python/fledge/services/core/scheduler/scheduler.py

```diff
@@ -128,6 +130,9 @@ def __init__(self):
     _storage = None
     _storage_async = None
 
+    # Startup tokens to pass to service or tasks being started
+    _startupTokens = dict()
```

Comment:
I believe this should be with service registry or core server.py and only for services (schedule type STARTUP)

Reply:
It would make more sense if there are created and maintained by the service registry since that will be doing the checking. It also gives us a single place to tighten up on the way they are used.